backend/pages/admin.py

```
@@ -2,11 +2,11 @@
 
 from .models import Quarter, ExcelFile, CSVData
 from unfold.admin import ModelAdmin
-
-
-#@admin.register(MyModel)
-#class CustomAdminClass(ModelAdmin):
-#    pass
+from django.contrib import admin
```

Ruff (0.11.9)

Redefinition of unused `admin` from line 1

Remove definition: `admin`

(F811)